File: app/src/main/java/com/irontec/axier/rxweather/utils/BindingUtils.java
package com.irontec.axier.rxweather.utils;

import android.databinding.BindingAdapter;
import android.widget.ImageView;

import com.bumptech.glide.Glide;

/**
 * Created by axier on 14/7/17.
 */

public class BindingUtils {

    @BindingAdapter({"bind:imageUrl"})
    public static void loadImage(ImageView view, String url) {
        Glide.with(view.getContext()).load(url).into(view);
    }

}

File: app/src/main/java/com/irontec/axier/rxweather/utils/BindingUtils.kt
package com.irontec.axier.rxweather.utils

import android.databinding.BindingAdapter
import android.widget.ImageView

import com.bumptech.glide.Glide

/**
 * Created by axier on 14/7/17.
 */

object BindingUtils {

    @JvmStatic @BindingAdapter("bind:imageUrl")
    fun loadImage(view: ImageView, url: String) {
        Glide.with(view.context).load(url).into(view)
    }

}
